### DISCUSSION TITLE
Want to open modal on button action

### DISCUSSION THREAD

**Discussion by rmitesh** [filamentphp/filament#6746] [Help]
I have list of cards, which contain title and button for an action.
![image](https://github.com/filamentphp/filament/assets/48554454/2fda7c1f-1d74-46c0-8c19-1489781f68b2)

I have created custom page, and where I used the card. and added filament button

```blade
<x-filament::button>
    <x-heroicon-o-plus class="w-5 h-5" />
</x-filament::button>
```
Now, I want to open a modal on click of "plus" icon with inputs.

And also, can we add Input field where Settings button is placed.

**Comment by aisuvro** [comment]
Hi @rmitesh.

It's simple.
Create a modal
![image](https://github.com/filamentphp/filament/assets/2842856/9cfcfdba-7d18-4b11-9469-7f7347700f1a)

Open the modal.
![image](https://github.com/filamentphp/filament/assets/2842856/fb75f8d2-efb6-4495-b9b1-ac5241270864)

**Reply by mra-narola** [reply]
@aisuvro Thanks mate, it's working.
can I connect this button with the Livewire component, with Filament Inputs?
